OpenLIFUTransducerTracker/OpenLIFUTransducerTracker.py

```diff
@@ -1455,11 +1459,24 @@ def onStartPhotoscanGenerationButtonClicked(self):
             except CalledProcessError as e:
                 slicer.util.errorDisplay("The underlying Meshroom process encountered an error.", "Meshroom error")
                 raise e
+            self.updatePhotoscanGeneratorProgressBar(new_photoscan_generator_progress_value = 100)
+            self.ui.photoscanGenerationStatusMessage.hide()
```

Comment:
this hide is not reached if we hit the `except` just above. I think you can fix this by putting these UI update lines inside a `finally` after the `try` and `except`